.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- HTML Comments can be left in place or removed. -->
+
+Proposed changes
+======
```

Comment:
I find the H1 overly large. Is it possible to drop them down to a smaller heading?

Reply:
How's that? If we all like this I'll sync the changes back to k3s.